recipes/libint2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "libint" %}
+{% set version = "2.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/evaleev/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tgz
+  sha256: 253fa3301e6742f5aac9e96dbe2a5707fc73abd6f4a214a8d44f03e113e69f00
+
+build:
+  number: 0
+  binary_relocation: true
```

Comment:
FYI - This is the default. So can be dropped.

Reply:
done